README.md

```diff
@@ -84,8 +80,16 @@ GuiHolder guiHolder = new GuiHolder(6, "Gui with number input", layout, yourPlug
 guiHolder.show(player);
 ```
 
+# Usage
+
+- Depend on [KotlinSpice](https://github.com/CultOfClang/KotlinSpice) in your plugin config, and request users to download
+the plugin to their server.
+- Or: Manage [shading](https://imperceptiblethoughts.com/shadow/) Kotlin. Shading Kotlin can cause hard-to-find errors if two different versions are present during runtime!
+
+**!! We are now using Github Packages which you can depend on similar to our other library, [Idofront](https://github.com/MineInAbyss/Idofront), but it is a bit more annoying, and JitPack will probably still work.**
+
 ## Use with Gradle
-```
+```groovy
```

Comment:
I'd like this to be updated for the Github Packages way eventually, but i've held off doing it for now since JitPack works okay, and I don't know how to do it with Maven.